flexneuart/models/utils.py

```diff
@@ -45,7 +45,7 @@ def init_model(obj_ref, bert_flavor : str,
 
     obj_ref.BERT_MODEL = bert_flavor
 
-    model = AutoModel.from_pretrained(bert_flavor)
+    model = AutoModel.from_pretrained(bert_flavor, trust_remote_code=True)
```

Comment:
I would rather not have it enabled by default (for security reasons). Could we have it as a parameter? Only one model (vanilla_bert_stand) need to use this paramter.